src/Libraries/RevitServices/Elements/ModelUpdater.cs

```diff
@@ -29,36 +29,50 @@ public class RevitServicesUpdater
 
         private readonly HashSet<IUpdater> registeredUpdaters;
 
+        [Obsolete("Use ElementsUpdated event")]
```

Comment:
Yes, all the known clients have been updated to subscribe to the new event. For compatibility we are keeping the old events.